OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoBriefingLogic.cs

```diff
@@ -29,7 +29,8 @@ public GameInfoBriefingLogic(Widget widget, ModData modData, World world)
 			var missionData = world.Map.Rules.Actors[SystemActors.World].TraitInfoOrDefault<MissionDataInfo>();
 			if (missionData != null)
 			{
-				var text = WidgetUtils.WrapText(missionData.Briefing?.Replace("\\n", "\n"), mapDescription.Bounds.Width, mapFont);
+				var briefing = missionData.Briefing != null ? previewWidget.Preview().GetLocalisedString(missionData.Briefing) : "";
+				var text = WidgetUtils.WrapText(briefing.Replace("\\n", "\n"), mapDescription.Bounds.Width, mapFont);
```

Comment:
`.Replace("\\n", "\n")` can be dropped here and in MissionBrowserLogic.cs since the newlines are directly in the translation file now.

Reply:
I thought of backwards compatibility with older maps, but missions probably need upgrades anyway.